data/858/632/99/85863299.geojson

```diff
@@ -28,6 +28,9 @@
     "name:spa_x_preferred":[
         "Poligono Industrial de Cogullada"
     ],
+    "name:spa_x_preferred_x_variant":[
```

Comment:
This one will be valid variant spelling – or we promote it to preferred and demote existing preferred to variant?